tox.ini

```diff
@@ -23,6 +23,7 @@ ignore_basepython_conflict = true
 commands =
     superset db upgrade
     superset init
+    superset load-test-users
```

Comment:
This order is different than the one specified by [Integration Testing](https://github.com/apache/superset/blob/master/CONTRIBUTING.md#integration-testing) in `CONTRIBUTING.md`. 

In the same file we have:

```
# Create default roles and permissions
superset init
```
Could you check if only the roles and permissions are created by `superset init` but not their associations with users? If the associations are also handled by `superset init` we would need to load the test users before that step.

Reply:
@michael-s-molina my original intent was to follow the order you outlined in the `CONTRIBUTING.md` (which is referenced in a few places), however CI fails with the following, 

```
self = <tests.integration_tests.sqllab_tests.TestSqlLab testMethod=test_query_admin_can_access_all_queries>

    def test_query_admin_can_access_all_queries(self) -> None:
        """
        Test query api with all_query_access perm added to
        Admin and make sure only Admin queries show up. This is the default
        """
        # Test search_queries for Admin user
        self.run_some_queries()
        self.login("admin")
    
        url = "/api/v1/query/"
        data = self.get_json_resp(url)
>       self.assertEqual(3, len(data["result"]))
E       AssertionError: 3 != 2
```

I looked into this previously, but couldn't find a resolution. I suspect this is just one of a magnitude of gremlins we have in the code base due to us not adhering to best practices. Previously the test users used to be loaded as part of the integration tests, i.e., after the `superset init` and I suspect there's likely some weird race condition.

Reply:
@michael-s-molina my original intent was to follow the order you outlined in the `CONTRIBUTING.md` (which is referenced in a few places), however CI fails with the following, 

```
self = <tests.integration_tests.sqllab_tests.TestSqlLab testMethod=test_query_admin_can_access_all_queries>

    def test_query_admin_can_access_all_queries(self) -> None:
        """
        Test query api with all_query_access perm added to
        Admin and make sure only Admin queries show up. This is the default
        """
        # Test search_queries for Admin user
        self.run_some_queries()
        self.login("admin")
    
        url = "/api/v1/query/"
        data = self.get_json_resp(url)
>       self.assertEqual(3, len(data["result"]))
E       AssertionError: 3 != 2
```

I looked into this previously, but couldn't find a resolution. I suspect this is just one of a magnitude of gremlins we have in the code base due to us not adhering to best practices. Previously the test users used to be loaded as part of the integration tests, i.e., after the `superset init` and I suspect there's likely some weird race condition.